scripts/publish-docs.js

```diff
@@ -16,10 +12,17 @@ if (process.env.TRAVIS === 'true') {
   shelljs.exec(`git config --global user.email "${userEmail}"`);
   shelljs.exec(`git config --global user.name "${userName}"`);
   shelljs.exec(`git remote add ${docsOrigin} ${remoteUrl} > /dev/null 2>&1`);
+} else {
+  // This part is in case you run the script locally, in which case you don't
+  // need the `GH_TOKEN` and can simply use the ssh remote.
+  const remoteUrl = 'git@github.com:commercetools/ui-kit.git';
+  shelljs.exec(`git remote add ${docsOrigin} ${remoteUrl} > /dev/null 2>&1`);
 }
 
-shelljs.exec(`git checkout ${docsBranch}`);
-shelljs.exec('git pull -r');
-// This assumes that git is checked out at this branch tag.
-shelljs.exec(`git merge ${process.env.TRAVIS_TAG}`);
-shelljs.exec(`git push --force-with-lease ${docsOrigin} ${docsBranch}`);
+// Make sure there is no existing docs branch locally
+shelljs.exec(`git branch -D ${docsBranch} > /dev/null 2>&1`);
+// Checkout a new local branch for `latest/docs`
+shelljs.exec(`git checkout -b ${docsBranch}`);
+shelljs.exec(`git push --force ${docsOrigin} ${docsBranch}`);
+// Switch back to the previous branch (e.g. `master`)
+shelljs.exec(`git checkout -`);
```

Comment:
Would it make sense to hard-code master? Do we need the dynamic nature of this? Can it happen that we want to switch back to anything other than master?

Reply:
I'll remove this actually, I think it just causes confusion